.github/workflows/node.js.yml

```diff
@@ -1,27 +0,0 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```

Comment:
Let's keep this workflow. Feel free to remove the build matrix.

Reply:
Do you want to run the linter and build everything just as a double-check in case somebody avoided the git hooks?